tests/sentry/api/endpoints/test_auth_index.py

```diff
@@ -39,9 +38,8 @@ def test_invalid_password(self):
         user = self.create_user("foo@example.com")
         response = self.client.post(
             self.path,
-            HTTP_AUTHORIZATION=u"Basic {}".format(
-                b64encode(u"{}:{}".format(user.username, "foobar"))
-            ),
+            HTTP_AUTHORIZATION=b"Basic "
+            + b64encode(u"{}:{}".format(user.username, "foobar").encode("utf-8")),
```

Comment:
Should we establish a preference for `force_text` versus encoding to utf-8? Or, just something to think about.

Reply:
IMO encoding to utf-8 is much more clear what's happening.

Reply:
For text for when you don't know exactly what you're getting.